ops/cloud-deployment/ustp-cams.bicep

```
@@ -52,6 +51,12 @@ param sqlServerResourceGroupName string = ''
 @description('Flag to enable Vercode access to execute DAST scanning')
 param allowVeracodeScan bool = false
 
+@description('Managed identity name with access to the key vault for Pacer Api credentials')
+param pacerKeyVaultIdentityName string
+
+@description('Resource group name managed identity with access to the key vault for Pacer Api credentials')
+param packerKeyVaultIdentityResourceGroupName string
```

```suggestion
param pacerKeyVaultIdentityResourceGroupName string
```

`packer` ==> `pacer`

---

I won't put suggestions on the use of this variable so that we are less likely to miss some because I may not see them all.

---

Did search on typo and replace all.